ci/docker/base.ml.Dockerfile

```diff
@@ -3,4 +3,5 @@ FROM $DOCKER_IMAGE_BASE_TEST
 
 COPY . .
 
+RUN BUILD=1 ./ci/ci.sh init
```

Comment:
this part is required to have nvm

Reply:
this file is from civ1. why this is required?

can nvm be installed via some other way in `data.build.Dockerfile`?

Reply:
it can yes, but we'll need to do the same for for ml.build.Dockerfile and rllib.build.Dockerfile

in civ1, nvm is installed twice, once in base.build.Dockerfile and once is build.Dockerfile; and the ml image used for data/ml/rllib is built through a layer of base.test < base.ml < build < ml which has a whole lot of useless stuff (if you look at base.ml and ml they are exactly the same)

civ2 ignore build.Dockerfile (which basically builds ray) and attempt to build ray as part of rayci itself; which requires the nvm to be installed; but putting nvm into wanda is nicer for caching so i just move it to the base itself.